priv/templates/phoenix.gen.json/controller.ex

```diff
@@ -49,7 +50,7 @@ defmodule <%= module %>Controller do
 
     # Here we use delete! (with a bang) because we expect
     # it to always work (and if it does not, it will raise).
-    <%= singular %> = Repo.delete!(<%= singular %>)
+    _<%= singular %> = Repo.delete!(<%= singular %>)
```

Comment:
Maybe we just should not assign it to anything? Or it is good because it names the return value?